packages/cloud/src/cloud/uploadFiles.ts

```diff
@@ -0,0 +1,94 @@
+import { ClientFile, uploadFile as uploadFileToPortive } from '@portive/client';
+import { Value } from '@udecode/plate-core';
+import { nanoid } from 'nanoid';
+import { CloudEditor, FileEvent, ProgressEvent } from './types';
+
+function createFileEvent(id: string, clientFile: ClientFile): FileEvent {
+  if (clientFile.type === 'image') {
+    return {
+      id,
+      type: 'image',
```

Comment:
we should use `getPluginType` anywhere `type` is used, so the consumer can customize it